app/styles/components/universal-selection.less

```diff
@@ -146,4 +146,9 @@
   .ember-power-select-options[role=listbox] {
     max-height: 400px!important;
   }
+
+  .description {
+    font-size: 16/@rem;
+    text-align: center;
+  }
```

Comment:
you can add classes to the element for that: there is a `text-size-X` that should match + `upf-align--center`